static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -1,4 +1,197 @@
+                position: absolute;
+                bottom: -$quote-tail - 2px;
+                left: $gs-gutter/2;
+                background-color: $neutral-garett-1;
+                border: solid 1px $neutral-5;
+                border-top: solid 1px $neutral-garett-1;
+            }
+            @include mq(mobileLandscape) {
+                left: $gs-gutter;
+            }
+            @include mq(desktop) {
+                left: $gs-gutter - $gs-baseline;
+            }
+            @include mq(leftCol) {
+                left: $gs-baseline;
+            }
+        }
+
+        .pullquote-cite,
+        .pullquote-paragraph {
+            display: inline;
+            font-weight: 100!important;
+        }
+
+        .inline-quote,
+        blockquote {
+            display: inline;
+        }
+        // Hack hide for svg
+        .closing {
+            display: none;
+        }
+    }
+    // Quote supporting
+    .element-pullquote.element--supporting {
+        @include mq(mobileMedium) {
+            clear: left;
+            float: left;
+            margin-right: $gs-gutter/2;
+            width: gs-span(3);
+        }
+        @include mq(leftCol) {
+            margin-left: - gs-span(3)/2 - $gs-gutter/2;
+        }
+
+        &:after {
+            @include mq(leftCol) {
+                border-radius: 0 0 0 $quote-tail;
+                left: gs-span(3)/2 - 28px;
+            }
+        }
+
+        .pullquote-cite,
+        .pullquote-paragraph {
+            @include fs-headlineGarnett(1);
+        }
+    }
+    // Quote halfWidth
+    .element-pullquote.element--halfWidth {
+        @include mq(mobileMedium) {
+            clear: left;
+            float: left;
+            margin-right: $gs-gutter/2;
+            width: gs-span(3);
+        }
+        @include mq(mobileLandscape) {
+            width: gs-span(4);
+        }
+
+        .pullquote-cite,
+        .pullquote-paragraph {
+            @include mq(mobile) {
+                @include fs-headlineGarnett(2);
+            }
+            @include mq(desktop) {
+                @include fs-headlineGarnett(3);
+            }
+        }
+    }
+    // Quote inline
+    .element-pullquote.element--inline {
+        @include mq(mobile) {
+            width: gs-span(4);
+        }
+        @include mq(mobileLandscape) {
+            width: gs-span(5);
+        }
+        @include mq(tablet) {
+            width: gs-span(6);
+        }
+
+        .pullquote-cite,
+        .pullquote-paragraph {
+            @include mq(mobile) {
+                @include fs-headlineGarnett(2);
+            }
+            @include mq(desktop) {
+                @include fs-headlineGarnett(3);
+            }
+        }
+    }
+    // Quote showcase
+    .element-pullquote.element--showcase {
+        @include mq(mobileLandscape) {
+            margin-left: -$gs-gutter;
+        }
+        @include mq(leftCol) {
+            width: gs-span(8);
+            margin-left: - gs-span(2);
```

Comment:
-gs-span(2); ?

Reply:
Chatted IRL, thinks it's part of the name without the space

Reply:
🌮 🎉